interfaces/builtin/utils.go

```diff
@@ -83,3 +83,15 @@ func udevUsbDeviceSnippet(subsystem string, usbVendor int64, usbProduct int64, k
 	udevSnippet.WriteString("\n")
 	return udevSnippet.Bytes()
 }
+
+// Function to support creation of udev TAG snippet for all the apps bound
+// to a given slot.
+func udevSecurityTagSnippet(deviceNode string, snapName string, apps map[string]*snap.AppInfo) []byte {
```

Comment:
As said before, there is no need for this function. The only thing we should move into its own function is the creation of the "snap_%s_%s" string as that one happens at different places.